### DISCUSSION TITLE
能否通过设置多个密钥来提高并发

### DISCUSSION THREAD

**Discussion by miantiao-me** [immersive-translate/immersive-translate#2079] [Ideas]
### 新功能 | New Feature

1. 设置多个翻译密钥，用于提高并发数。

### 为什么您想要添加这些功能 | Why you want to add these features

1. 大模型有 QPM 限制，不同账户的多个 Key 可以提高限制

**Comment by theowenyoung** [comment]
用 `,`分隔多个key 即可，目前已支持。
